projects/ccd-case-ui-toolkit/src/lib/shared/components/case-editor/case-edit/case-edit.component.ts

```diff
@@ -409,7 +409,7 @@ private caseSubmit({ form, caseEventData, submit }: CaseEditCaseSubmit ): void {
         this.sessionStorageService.removeItem('eventUrl');
         const confirmation: Confirmation = this.buildConfirmation(response);
         if (confirmation && (confirmation.getHeader() || confirmation.getBody())) {
-          this.confirm(confirmation);
+          this.confirm(confirmation).then().catch();
```

Comment:
@DavidJayakumar - empty `catch()` clause? If intentional, please consider adding a comment to explain